physlr/physlr.py

```diff
@@ -1726,19 +1726,32 @@ def subgraph_stats_process(u):
         return Physlr.subgraph_stats(Physlr.graph, u)
 
     def physlr_subgraphs_stats(self):
-        "Retrieve subgraphs' stats."
-        gin = self.read_graph(self.args.FILES)
-        Physlr.filter_edges(gin, self.args.m)
-        print(
-            int(timeit.default_timer() - t0),
-            "Computing statistics of the subgraphs...", file=sys.stderr)
+        "Retrieve statistics of subgraphs."
```

Comment:
Could we just add a small 'usage' comment? (similar to physlr_map, for example)

Reply:
I just added more detail. But its a side utility; it is not used in the main pipeline.

Reply:
But you could still call `physlr subgraph-stats` right? (even if it's not in the main pipeline)
Just thinking if any of us want to do it in the future it would help to know the expected arguments
```
Usage: physlr subgraph-stats <graph .tsv> [TSV file listing barcodes]
```
(or whatever format the TSV argument is)

Reply:
Oh I see, I thought you mean at the beginning of this function. I'll just do it.

Reply:
Aha, Done